samples/wcfapp/Dockerfile.client

```diff
@@ -0,0 +1,9 @@
+FROM microsoft/dotnet-framework:4.7.2-sdk AS build
+WORKDIR /app
+COPY WcfClient/. .
+RUN msbuild /p:Configuration=Release
+
+FROM microsoft/dotnet-framework:4.7.2-runtime AS runtime
+WORKDIR /app
+COPY --from=build /app/bin/Release .
+CMD WcfClient.exe
```

Comment:
I wonder if this shouldn't be set as the ENTRYPOINT instead.  It is the only purpose of the image.  Same comment applies to `Dockerfile.console`